Libraries/xcdriver/Sources/HelpAction.cpp

```diff
@@ -0,0 +1,221 @@
+/**
+ Copyright (c) 2016-present, Facebook, Inc.
+ All rights reserved.
+
+ This source code is licensed under the BSD-style license found in the
+ LICENSE file in the root directory of this source tree. An additional grant
+ of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#include <xcdriver/HelpAction.h>
+#include <xcdriver/UsageAction.h>
+#include <libutil/FSUtil.h>
+#include <libutil/SysUtil.h>
+
+using xcdriver::HelpAction;
+using libutil::FSUtil;
+using libutil::SysUtil;
+
+HelpAction::
+HelpAction()
+{
+}
+
+HelpAction::
+~HelpAction()
+{
+}
+
+int HelpAction::
+Run()
+{
+    UsageAction::Run();
+
+    fprintf(stdout, "Options:\n");
+    fprintf(
+        stdout,
+        "    -usage                                      "
+        "print brief usage instructions\n");
+    fprintf(
+        stdout,
+        "    -help                                       "
+        "print complete usage instructions\n");
+    fprintf(
+        stdout,
+        "    -verbose                                    "
+        "print diagnostic messages and other detailed information when "
+        "performing an action\n");
+    fprintf(
+        stdout,
+        "    -license                                    "
+        "print xcbuild's license (not yet implemented)\n");
```

Comment:
nit: Might say just "print license" rather than hardcoding the name. (Should file a starter task for this one too, would be pretty easy to add!)